rest-service/manager_rest/resource_manager.py

```diff
@@ -492,25 +495,30 @@ def _update_finished_execution_dependencies(self, execution):
         if execution._deployment_fk:
             deployment = execution.deployment
         else:
-            return
+            return None, None
 
         workflow_id = execution.workflow_id
         if workflow_id == 'delete_deployment_environment':
-            return
+            return None, None
 
         try:
             update_inter_deployment_dependencies(self.sm, deployment)
         except Exception as e:
             now = datetime.utcnow()
+            error_message = (
+                'Failed updating dependencies of deployment '
+                f'{deployment.id}: {e}'
+            )
             new_log = models.Log(
                 reported_timestamp=now,
                 timestamp=now,
                 execution=execution,
-                message=f'Failed updating dependencies of deployment '
-                        f'{deployment.id}: {e}'
+                message=error_message,
+                level='error',
```

Comment:
(but no need on this PR)

Reply:
Yeah, would be good, but I'm not even sure i want to put more new non-nullables in 7.0 - we really need new snapshots (and no old snapshots) so that we can stop worrying about this shit :P